lib/jscad-fns/rotate.tsx

```diff
@@ -0,0 +1,8 @@
+export type RotateProps = {
+  angles: [number, number, number]
```

Comment:
see Translate for more details

```suggestion
  angles: Point3
```

Reply:
I see that Translate has args: [number, number, number]. 
How should I fix the issue there?